lib/ex_machina/ecto.ex

```diff
@@ -66,6 +70,34 @@ defmodule ExMachina.Ecto do
     |> drop_ecto_fields
   end
 
+  @doc """
+  Same as `params_for/2`, but it inserts all associations and sets the
+  foreign_keys.
+
+  ## Example
+
+      def article_factory do
+        %MyApp.Article{title: "An Awesome Article", author: build(:user)}
```

Comment:
Maybe `build(:author)` so the terminology remains the same throughout the example?